crates/ironrdp/examples/server.rs

```diff
@@ -230,7 +235,13 @@ struct StubSoundServerFactory {
 
 impl ServerEventSender for StubSoundServerFactory {
     fn set_sender(&mut self, sender: UnboundedSender<ServerEvent>) {
-        let mut inner = self.inner.lock().unwrap();
+        let mut inner = match self.inner.lock() {
+            Ok(inner) => inner,
+            Err(e) => {
+                warn!("Failed to acquire the mutex: {e:?}");
+                return;
```

Comment:
suggestion: It’s fine to panic on mutex poisoning, it means we already panicked somewhere else and we are merely propagating it

`.expect("poisoned")`